Sketch.java

```diff
@@ -8,29 +8,93 @@ public class Sketch extends PApplet {
    */
   public void settings() {
 	// put your size call here
-    size(400, 400);
+    size(400, 500);
   }
 
   /** 
    * Called once at the beginning of execution.  Add initial set up
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    background(50, 50, 200);
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
-	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
 
-    stroke(255);
-    line(50, 125, 70, 50);  
+    house();
+    sun(400,0);
+    sign();
+    grass();
```

Comment:
- need to make multiple calls to methods to demonstrate reusability.  
- Should have more than one method defined with parameters.
- no method with return values defined